### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by gaurangkeskar**

### CODE REVIEW

File: headers/Reader.h
Comment (cctechyashdoshi) [inline headers/Reader.h:1]: You can remove the "using namespace std;" as it's not needed

File: src/Reader.cpp
Comment (cctechyashdoshi) [inline src/Reader.cpp:1]: the code seems hard coded, especially this part of code👇

data += line.substr(getPos + 7);

**Review by cctechyashdoshi — CHANGES_REQUESTED** [review]
I have added some corrections, please look through.

File: headers/Reader.h
Comment (gaurangkeskar) [reply headers/Reader.h:1]: reader.cpp will throw errors if i remove it.

**Pull request merged**